src/components/addBook.js

```diff
@@ -1,18 +1,35 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <div className="mx-auto m-4">
-    <h2>ADD NEW BOOK</h2>
-    <form>
-      <input type="text" placeholder="Book title" />
-      <select type="select">
-        <option value="test">Lorem 1</option>
-        <option value="test">Lorem 2</option>
-        <option value="test">Lorem 3</option>
-      </select>
-      <input type="submit" value="Add Book" />
-    </form>
-  </div>
-);
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const handleClick = () => {
+    const eTitle = document.getElementById('title');
+    const eCategory = document.getElementById('category');
+    const bookTitle = eTitle.value;
+    const bookCategory = eCategory.options[eCategory.selectedIndex].text;
+    const id = uuidv4();
+    eTitle.value = '';
+    dispatch(addBook({ title: bookTitle, category: bookCategory, id }));
+  };
+
+  return (
+    <div className="mx-auto m-4">
+      <h2>ADD NEW BOOK</h2>
+      <div>
+        <input type="text" id="title" placeholder="Book title" />
+        <select type="select" id="category">
+          <option value="Lorem 1">Lorem 1</option>
+          <option value="Lorem 2">Lorem 2</option>
+          <option value="Lorem 3">Lorem 3</option>
```

Comment:
- [ ] The functionality to **Add** and **Remove** a book doesn't work. Please check and fix it.